work2/myworld/members/views.py

```diff
@@ -0,0 +1,64 @@
+from django.shortcuts import render
+from django.views import View
+from .models import Employees
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+
+@method_decorator(csrf_exempt, name='dispatch')
+class EmployeeView(View):
+    def get(self, request, empid=None, department=None):
+        employee_model_list = []
+        try:
+            if empid:
+                employee_model_list = Employees.objects.filter(emp_id=empid)
+            elif department:
+                employee_model_list = Employees.objects.filter(department=department)
+        except Employees.DoesNotExist:
+            return JsonResponse({'status': 'failed', "employees": None}, status=400)
+        employees = []
+        for employee in employee_model_list:
+            data = {
+                    "first_name" : employee.first_name,
+                    "last_name": employee.last_name,
+                    "address": employee.address,
+                    "emp_id": employee.emp_id,
+                    "salary": employee.salary,
+                    "department": employee.department
+                }
+            employees.append(data)
+            return JsonResponse({'status': 'success', "employees": employees}, status=200)
+    
+    def post(self, request):
+        if not request.POST.get('first_name') or not request.POST.get('last_name') or not request.POST.get('address') or  not request.POST.get('emp_id') or not request.POST.get('salary'):
+            return JsonResponse({'status': 'failed', "message" : "all fields required"}, status=500)
+        Employees.objects.create(first_name= request.POST.get('first_name'),
+            last_name= request.POST.get('last_name'),
+            address= request.POST.get('address'),
+            emp_id= request.POST.get('emp_id'),
+            salary= request.POST.get('salary'),
+            department= request.POST.get('department'))
+        return JsonResponse({'status': 'sucess'}, status=200)
+    def update(self,request,empid=None):
+        try:
+            employee = Employees.objects.get(emp_id=empid)
+        except Employees.DoesNotExist:
+            return JsonResponse({'status': 'failed', "employees": None}, status=400)
```

Comment:
@sampleharvgithub The response message what you're returning here, does it give any useful information for the client who made the API call?